client/src/components/pages/database/license-availability-table.tsx

```diff
@@ -10,9 +33,38 @@ const LicenseAvailabilityTable = () => {
     'Total Licenses'
   ]
 
+
+  
+  const formatData = (data: BusinessLicense[], zipcodeList: Set<EligibleBostonZipcode>) => {
```

Comment:
One small code comment:
This function should be defined outside of the react component. There is a tiny performance impact where it will be redeclared every time the component is updated. Though the real benefit of defining it outside is clearer separation of concerns and easier testing. ^^